README.md

```diff
@@ -6,47 +6,30 @@ direnv -- unclutter your .profile
 [![latest packaged version(s)](https://repology.org/badge/latest-versions/direnv.svg)](https://repology.org/project/direnv/versions)
 [![Support room on Matrix](https://img.shields.io/matrix/direnv:numtide.com.svg?label=%23direnv%3Anumtide.com&logo=matrix&server_fqdn=matrix.numtide.com)](https://matrix.to/#/#direnv:numtide.com)
 
-`direnv` is an extension for your shell. It augments existing shells with a
-new feature that can load and unload environment variables depending on the
-current directory.
+`direnv` is an extension for your unix shell. It augments existing shells with a
+new feature that can **load and unload environment variables** depending on the
+directory you are in.
 
 ## Use cases
 
-* Load [12factor apps](https://12factor.net/) environment variables
+Primary use-cases **hacking on nix based software**:
 * Create per-project isolated development environments
-* Load secrets for deployment
+  * **flake support**: automates running **nix develop** when entering the project directory
+  * **nix-shell** support: automates running **nix-shell** when entering the project directory
 
-## How it works
+Generic use-case (no nix involved): **environment variable management**:
+* Load project specific variables for build, development and deployment
+  * Load [12factor apps](https://12factor.net/) environment variables
 
-Before each prompt, direnv checks for the existence of a `.envrc` file (and
-[optionally](man/direnv.toml.1.md#codeloaddotenvcode) a `.env` file) in the current
-and parent directories. If the file exists (and is authorized), it is loaded
-into a **bash** sub-shell and all exported variables are then captured by
-direnv and then made available to the current shell.
+Supported platforms are:
+* Unix-like operating system (Linux, macOS, Windows via WSL2, FreeBSD)
 
-It supports hooks for all the common shells like bash, zsh, tcsh and fish.
-This allows project-specific environment variables without cluttering the
-`~/.profile` file.
+Supported shells are:
+* **bash, zsh, fish, tcsh, csh, elvish, rc, powershell, murex, nushell and xonsh** shells.
 
-Because direnv is compiled into a single static executable, it is fast enough
-to be unnoticeable on each prompt. It is also language-agnostic and can be
-used to build solutions similar to rbenv, pyenv and phpenv.
-
-## Getting Started
-
-### Prerequisites
+Note: **direnv is limited to interactive shells** so 'ssh nixos@nixos "cd myproject; export" will not show added variables but 'ssh nixos@nixos' and then manually typing 'export' will show them.
```

Comment:
But you can use `ssh nixos@nixos "cd myproject; direnv exec . export"`

Reply:
That is good to know but I'd consider this a workaround or hack rather than 'supported'.